content/ja/docs/reference/access-authn-authz/rbac.md

```diff
@@ -0,0 +1,1109 @@
+---
+title: RBAC認可を使用する
+content_type: concept
+aliases: [/rbac/]
```

Comment:
I recommend removing this alias, as otherwise the Japanese and English localizations will conflict.

Reply:
Thank you, I removed this alias.